core/state_processor.go

```diff
@@ -78,7 +78,11 @@ func (p *StateProcessor) Process(block *types.Block, parent *types.Header, state
 	)
 
 	// Configure any stateful precompiles that should go into effect during this block.
-	p.config.CheckConfigurePrecompiles(new(big.Int).SetUint64(parent.Time), block, statedb)
+	err := p.config.ConfigurePrecompiles(new(big.Int).SetUint64(parent.Time), block, statedb)
+	// ASK: Should we panic instead?
+	if err != nil {
+		return nil, nil, 0, fmt.Errorf("could not configure precompiles: %w", err)
+	}
```

Comment:
We don't NEED to panic, but an error here would indicate that the subnet is most likely borked.

Could we change this to log an error and not panic? If a subnet ends up borked for this reason, it will start to emit error logs, but I think that's better than panic-ing.

Reply:
added logs